docs/services/data/train-model.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Train a Model"
+linkTitle: "Train Model"
+weight: 40
+type: "docs"
+tags: ["data management", "ml", "model training"]
+description: "Train an image classification model on labeled image data."
+# SME: Aaron Casas
+---
+
+You can tag [images collected](../configure-data-capture) by robots and use the labeled data to train a **Single Label** or **Multi Label** image classification model within Viam.
+
+## Label a dataset
+
+To label a dataset, go to the [**DATA** tab](https://app.viam.com/data/view) in the Viam app.
+
+![Add new model menu](../img/add-new-model.png)
+
+On the **IMAGES** sub-tab, you can filter available images, using the **FILTERING** menu and select the attributes that match where, how, and when the data was collected.
+
+{{< alert title="Important" color="note" >}}
+Datasets are views, not materialized.
+If the underlying data matching the filter changes because data is deleted or more data is added, the dataset will also change.
+
+Therefore the data you are viewing may change as you label and train on the dataset.
+{{< /alert >}}
+
+If you would like to create a model that identifies an image of a star in a set of images, tag each image with `star` or `notstar` respectively.
```

Comment:
nit: You don't have to label `notstar`, since we will automatically add a label of `UNKNOWN` to any image that is unlabeled. 

It's useful to tag `notstar` if you then want to filter down the data further on the left side (e.g. add `star` and `notstar`) so that you only train on the images that you've specifically labeled.